src/dlgTriggerEditor.cpp

```diff
@@ -8370,15 +8365,50 @@ void dlgTriggerEditor::resizeEvent(QResizeEvent* event)
 void dlgTriggerEditor::slot_key_grab()
 {
     mIsGrabKey = true;
+    setShortcuts(true);
+    QCoreApplication::instance()->installEventFilter(this);
+}
+
+// Activate shortcuts for editor menu items like Ctrl+S for "Save Item" etc.
+// Deactivate instead with optional "true" - useful to grab for a keybinding
```

Comment:
It is not just "useful" - it is ***essential*** to deactivate shortcuts when grabbing a keybinding!

Reply:
I searched 30min+ until #350 explained me why this is happening in code, so I added a comment.